draft-ietf-httpbis-http2-encryption.md

```diff
@@ -192,16 +193,20 @@ Given that the primary goal of this specification is to prevent passive attacks,
 critical failings (especially considering the alternative - HTTP over cleartext). However, a modest
 form of protection against active attacks can be provided for clients on subsequent connections.
 
-When an origin is able to commit to providing service for a particular origin over TLS for a
-bounded period of time, clients can choose to rely upon its availability, failing when it cannot be
+When an origin is able to commit to providing service for a particular origin over TLS for a bounded
+period of time, clients can choose to rely upon its availability, failing when it cannot be
 contacted. Effectively, this makes the choice to use a secured protocol "sticky".
 
 
 ## Opportunistic Commitment
 
-An insecure origin can commit to providing a secured alternative by including a `commit` member in
-the http-opportunistic well-known resource (see {{well-known}}), whose value is a number
-representing an interval in seconds.
+An origin can reduce the risk of attacks on opportunistically secured connections by committing to
+provide an secured, authenticated alternative service. This is done by including the optional
+`commit` member in the http-opportunistic well-known resource (see {{well-known}}). This feature is
+optional due to the requirement for server authentication and the potential risk entailed (see
+{{pinrisks}}).
+
+The value of the `commit` member is the duration of the commitment interval in seconds.
```

Comment:
Saying it's a number clarifies what JSON is expected; perhaps "JSON number [ref]"